src/builtins/bundle-url.js

```diff
@@ -22,7 +22,7 @@ function getBundleURL() {
 }
 
 function getBaseURL(url) {
-  return ('' + url).replace(/^((?:https?|file|ftp):\/\/.+)\/[^\/]+$/, '$1') + '/';
+  return ('' + url).replace(/^((?:https?|file|ftp):\/\/.+)\/[^/]+$/, '$1') + '/';
```

Comment:
Here too. ESLint seems to have removed some bit of regex. Again, ignore if I'm wrong. 😅

Reply:
Same here. The \ was an unnecessary character escape. Same functionality.